jsonnet/openshift-state-metrics.libsonnet

```diff
@@ -13,11 +13,21 @@ function(params) {
     },
   },
 
+  deployment: osm.openshiftStateMetrics.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/name': 'openshift-state-metrics',
+        'app.kubernetes.io/component': 'exporter',
+        'app.kubernetes.io/part-of': 'openshift-monitoring',
```

Comment:
Same as the other comments, I think we should only keep:
```
'app.kubernetes.io/managed-by': 'cluster-monitoring-operator'
'app.kubernetes.io/part-of': 'openshift-monitoring'
```
And propagate `'app.kubernetes.io/part-of'` from the main config. The rest should be added directly in the project's repo.

Reply:
This suggested change require addition of 'commonLabels' to the the upstream project openshift-state-metrics. WDYT ?

Reply:
Yes, I think that would be the best approach

Reply:
PTAL https://github.com/openshift/openshift-state-metrics/pull/75